src/data/MmXsmmFused.hpp

```diff
@@ -211,11 +262,26 @@ class edge::data::MmXsmmFused< float > {
         delete[] l_rows; delete[] l_cols; delete[] l_vals;
       }
       else {
-        m_kernels[i_group].push_back( libxsmm_create_rm_ac_soa( m_descs[i_group].back() ).smm );
+        m_kernels[i_group].push_back( libxsmm_create_pgemm_ac_rm( m_descs[i_group].back(), PP_N_CRUNS ).smm );
```

Comment:
There's a non-preprocessor version of `PP_N_CRUNS` called `N_CRUNS`: https://github.com/3343/edge/blob/develop/src/constants.hpp#L285

Reply:
I used PP_N_CRUNS as this is used in the constructor. Should we also change the constructor then?

Reply:
No, guess it's fine for now.